File: task1/app/src/main/java/com/example/task1/MainActivity (2).java
package com.example.task1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {


    override fun onCreate(Bundle savedInstanceState) {
        val btn1: Button=findViewById(R.id.button5)

        super.onCreate(savedInstanceState);
        setContentView(R.layout.linearlayout);

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater inflater=getMenuInflater();
        inflater.inflate(R.menu.task1, menu);
        return true;
    }



}

File: task1/app/src/main/java/com/example/task1/MainActivity (2).kt
package com.example.task1

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.os.PersistableBundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.relativelayout)
        val btn1:Button=findViewById(R.id.button5)
        btn1.setOnClickListener {
        viewContact()

        }
    }
    fun viewContact(contactUri: Uri) {
        val intent = Intent(Intent.ACTION_VIEW, contactUri)
        if (intent.resolveActivity(packageManager) != null) {
            startActivity(intent)
        }
    }
}